internal/services/appservice/windows_web_app_resource_test.go

```diff
@@ -2703,7 +2703,7 @@ resource "azurerm_windows_web_app" "test" {
           count             = 1
           status_code_range = 500
           sub_status        = 30
-          win32_status      = 0
+          win32_status_code = 0
```

Comment:
should we add a linux test too?

Reply:
Steve has noted internal he will handle this comment later